chapters/en/Unit 1 - Fundamentals/Chapter 1 - Fundamentals of Computer Vision/applications.mdx

```diff
@@ -1,6 +1,10 @@
-# A high overview of computer vision system with examples
+# Applications Of Computer Vision
 
-In today's modern world computer vision is performing really difficult tasks which are even difficult for humans to do it. For example driving a 2 wheeler without helmet is a really difficult dangerous and can cause severe injuries to the person, now non helmet vehicle is a really tidious task as a human, this where computer vision comes into picture, it is an efficient approach to use object detection in detecting the non helemet vehicles. It has numerous applications in the real world. Here are a couple of examples:
+In today's modern world, computer vision systems perform challenging tasks that are even difficult for humans to do. Consider India, a country with the highest number of registered two-wheelers in the world. Some people forget using a helmet here which can be dangerous and can cause severe injuries to the person; to address these issues, the Government of India, in collaboration with other institutions, developed  Computer Vision systems which can automatically catch riders without a helmet along with their licence plate to impose strict laws against them.
+
+Computer Vision systems have numerous applications in Healthcare, Retail, Industries and many more. Here are a few high-level examples of computer vision systems:
+
+## A high overview of computer vision system with examples

```

Comment:
Just created a heading "Applications of Computer vision" and restructured aman's definition to be more specific in the context of how its is applied in India.